functional_testing/Grid_API_Testing/api_testing/testcases/basic_tests/test09_gateway_apis.py

```diff
@@ -205,19 +205,27 @@ def setUp(self):
                 {
                     "name": "private",
                     "type": "vxlan",
-                    "id": self.private_vxlan_id,
+                    "id": str(self.private_vxlan_id),
                     "config": {
                         "cidr": "192.168.2.20/24"
-                    }
+                    },
+                    "dhcpserver": {
```

Comment:
To use add dhcp api the nic should have dhcp term , and withoudt it the reponse will returns interface < nic name > doesn't have dhcp